### PR TITLE
Complete documentation for custom HTML5 params

### PR DESCRIPTION
**Pull request by matiasilva**
Hi everyone,

As promised in my [other PR](https://github.com/bigbluebutton/bigbluebutton/pull/9828), I've updated and completed the documentation on the custom HTML5 parameters. This is a massively useful tool and could great simplify the process for those who simply want to change the minor look and feel of the client. This documentation was started by @antobinary but in the meantime it is likely that more pressing issues came to light. Once these changes are approved, it would be great if this could be merged with the existing documentation and updated as soon as possible.

In a nutshell, this PR:

* organizes the custom params into easy-to-read tables
* implements user-friendly navigation with clear topics
* adds a reference to the FAQs for branding

For more information, read each commit message.

### CODE REVIEW

File: _posts/2019-02-14-customize.md [hunk 1]
```diff
@@ -907,125 +909,112 @@ $ yq w -i $TARGET public.app.clientTitle "New Title"
...
 
-If set to `true`, the current user's audio will be able to be muted/unmuted from the "parent" web page -- the same page where the BigBlueButton client is embedded
+### Layout parameters
 
+| Parameter                                  | Description                                                                                                           | Default value |
+| ------------------------------------------ | --------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `userdata-bbb_auto_swap_layout=`           | If set to `true`, the client will display the ask for feedback screen on logout                                       | `false`       |
+| `userdata-bbb_hide_presentation=`          | If set to `true`, the presentation area will be minimized automatically                                               | `false`       |
+| `userdata-bbb_show_participants_on_login=` | If set to `true`, the client will start the process of joining the audio bridge automatically upon loading the client | `false`       |
```
Comment (mtsonline) [inline _posts/2019-02-14-customize.md:984]: the description does not fit here

Comment (matiasilva) [reply _posts/2019-02-14-customize.md:984]: Well spotted!

Comment (matiasilva) [reply _posts/2019-02-14-customize.md]: I've added a fix for this. Thanks for pointing this out.

**Comment by matiasilva** [comment]
There's one more parameter that I've not yet fixed but I can't seem to understand what it does. If anyone knows what `bbb_auto_swap_layout` let me so I can update it.

**Comment by KDSBrowne** [comment]
I think it's used to determine if the presentation area should be minimized or not when a user joins a meeting.

File: _posts/2019-02-14-customize.md [hunk 2]
```diff
@@ -907,125 +909,112 @@ $ yq w -i $TARGET public.app.clientTitle "New Title"
...
 
-If set to `true`, the presentation area will be minimized automatically
+### Whiteboard parameters
 
-* `userdata-bbb_listen_only_mode=false`
-(default value is `false`)
+| Parameter                           | Description                                                                                                    | Default value |
+| ----------------------------------- | -------------------------------------------------------------------------------------------------------------- | ------------- |
+| `userdata-bbb_multi_user_pen_only=` | If set to `true`, the whiteboard will be multi-user only, meaning that all participants can annotate it freely | `false`       |
+| `userdata-bbb_presenter_tools=`     | If set to `false`, the client will display the presenter tools                                                 | `true`        |
```
Comment (jfsiebel) [inline _posts/2019-02-14-customize.md:967]: In fact this parameter enable only the pen tool for multi-whiteboard users.

Comment (jfsiebel) [inline _posts/2019-02-14-customize.md:967]: You can pass an array with the tools that you want to be available to the presenter.

File: _posts/2019-02-14-customize.md [hunk 3]
```diff
@@ -907,125 +909,112 @@ $ yq w -i $TARGET public.app.clientTitle "New Title"
...
+| `userdata-bbb_multi_user_tools=`    | If set to `true`, the client will display the multi-user tools                                                 | `false`       |
```
Comment (jfsiebel) [inline _posts/2019-02-14-customize.md:968]: Same thing as `bbb_presenter_tools` but for non presenters when multi-whiteboard is active.

**Comment by matiasilva** [comment]
Added the latest recommendations. Thanks @jfsiebel and @KDSBrowne.

**Comment by jfsiebel** [comment]
`userdata-bbb_multi_user_pen_only` is still incorrect.

**Comment by matiasilva** [comment]
Pesky line! Thanks and fixed.

**Comment by matiasilva** [comment]
Perfect! Last step needed is to merge the html5-custom-parameters branch into master.

**Comment by ffdixon** [comment]
Done!
